부산대_9조FE_링크나무_9주차

SeonghyeonKim opened the pull request:
## 9주 차 진행 사항

- 임시보관함과 카드에 드래그 기능 추가 (#61 )
- 드래그로 북마크 이동이 되도록 설정 (#63 )
- 모달에 사용하는 컴포넌트와 커스텀 훅 추가 (#57  )
- global modal로 모달 관리하도록 변경
- 노션 연동 기능 (#58 )
- 카카오톡 가져오기 기능 (#46 )
- 북마크 추가하기 기능 (#52 )
- 메뉴바 모달화 (#60 )
- 메뉴바의 카테고리 추가 버튼을 워크스페이스 추가 버튼으로 변경

<br>

## 10주 차 계획
- 구글 Docs 연동
- 프로젝트 배포
- API 명세서에 업데이트된 에러코드 반영
- 확장프로그램 제작
- 버그 수정
- 컴포넌트 별 디자인 수정

<br>

dabinchiii committed:
feat(store): 액세스 토큰을 반환하는 함수의 이름 변경

- getToken에서 getAccessToken으로 이름 변경

dabinchiii committed:
feat(route): 카테고리 선택 시 쿼리 매개변수를 이용한 동적 라우팅 구조로
변경

- 카테고리를 선택하면 현재 워크스페이스 id와 현재 카테고리 id를
  매개변수로 하여 동적 라우팅되도록 변경
- access token이 변경되면 데이터 refetch

dabinchiii committed:
Merge pull request #128 from SeonghyeonKim/fix/drag-card

드래그 카드에 한글의 길이가 범위를 넘는 경우, 카드 범위를 넘는 현상 수정

dabinchiii committed:
Merge pull request #11 from dabinchiii/fix/reload

Fix/reload

SeonghyeonKim committed:
Merge pull request #129 from dabinchiii/fix/reload

새로고침 시 현재 카테고리 유지되도록 변경

SeonghyeonKim committed:
Merge pull request #131 from Step3-kakao-tech-campus/weekly

배포 테스트를 위한 PR

SeonghyeonKim committed:
Merge pull request #132 from Step3-kakao-tech-campus/develop

배포 테스트를 위한 PR

dabinchiii committed:
Merge branch 'weekly' of https://github.com/Step3-kakao-tech-campus/Team9_FE into newWeekly

dabinchiii committed:
refactor(store): state.workspace.workspaceList를 가져오는 함수 변경

- state.workspace를 반환하는 함수를 제거하고 state.workspace.workspaceList를
  반환하는 함수 추가 및 적용

dabinchiii committed:
feat(hook): 워크스페이스/카테고리 ID로 워크스페이스/카테고리 이름을 찾는
커스텀 훅 추가

dabinchiii committed:
fix(hook): 워크스페이스 리스트를 가져오는 커스텀 훅 추가

dabinchiii committed:
fix(refetch): 워크스페이스, 카테고리 추가 후 데이터 다시 가져오는 기능

dabinchiii committed:
feat: prettier 적용 및 사용하지 않는 코드 삭제

dabinchiii committed:
fix(refetch): 노션 연동 성공 후 워크스페이스 리스트 refetch 기능

dabinchiii committed:
fix(refetch): 구글 docs 연동 성공 후 워크스페이스 리스트 refetch 기능

dabinchiii committed:
fix(refetch): 공유 링크로 워크스페이스/카테고리 추가하기 후 워크스페이스
리스트 refetch 기능

dabinchiii committed:
fix(refetch): 워크스페이스/카테고리 이름 변경 후 워크스페이스 리스트
refetch 기능

dabinchiii committed:
fix(refetch): 워크스페이스/카테고리 삭제 후 워크스페이스 리스트 refetch
기능

dabinchiii committed:
feat(hook): 불필요한 콘솔 출력 제거

dabinchiii committed:
Merge branch 'weekly' into fix/workspace-list

SeonghyeonKim committed:
fix: 토스트 메세지가 닫히는 시간 설정

SeonghyeonKim committed:
fix: 토스트 함수 변경

SeonghyeonKim committed:
fix: 이동 실패 시, 임시보관함 변경되지 않게 수정

SeonghyeonKim committed:
fix: 토스트 메세지 이후 새로고침 되도록 변경

SeonghyeonKim committed:
Merge pull request #133 from dabinchiii/fix/workspace-list

워크스페이스 리스트 데이터 변경 후 바로 refetch하는 기능

SeonghyeonKim committed:
fix: 카드 클릭 시 새창 열기

dabinchiii committed:
Merge pull request #135 from SeonghyeonKim/fix/move-bookmark

북마크 이동 후 새로고침 되도록 변경

SeonghyeonKim committed:
Merge pull request #136 from Step3-kakao-tech-campus/weekly

배포 테스트를 위한 PR

SeonghyeonKim committed:
Merge pull request #137 from Step3-kakao-tech-campus/develop

배포 테스트를 위한 PR

dabinchiii committed:
Merge branch 'weekly' of https://github.com/Step3-kakao-tech-campus/Team9_FE into newWeekly

dabinchiii committed:
Merge pull request #12 from dabinchiii/newWeekly

New weekly

dabinchiii committed:
fix(share): 공유 링크에서 암호화된 ID 추출 시 에러 수정

dabinchiii committed:
fix: 페이지 이동 시 useNavigate 사용하도록 변경

- window.location.href를 사용하여 이동하던 로직을 useNavigate를
  사용하도록 변경
- 이에 따라 북마크 그리드의 북마크들이 바로 변할 수 있도록 변경

SeonghyeonKim committed:
Merge pull request #140 from dabinchiii/fix/location

공유 링크에서 ID 추출 관련 에러 수정 및 페이지 이동 방식 변경

dabinchiii committed:
feat(page): 검색 결과 페이지 라우트 추가

- /search/result에 검색 결과 페이지 연결

SeonghyeonKim committed:
feat: 카드 컨텍스트 메뉴 컴포넌트 제작

SeonghyeonKim committed:
feat: 카드 컴포넌트에 우클릭 핸들러 추가

SeonghyeonKim committed:
feat: 북마크 삭제 타입 추가

SeonghyeonKim committed:
feat: 북마크 삭제 모달 컴포넌트 제작

SeonghyeonKim committed:
feat: 북마크 삭제 모달 추가

SeonghyeonKim committed:
feat: 북마크 수정 타입 추가

SeonghyeonKim committed:
feat: 북마크 수정 모달 컴포넌트 제작

SeonghyeonKim committed:
feat: 북마크 수정 모달 추가

dabinchiii committed:
feat(GNB): 공유 페이지에 사용되던 GNB를 기본 GNB로 사용할 수 있도록 변경

- 컴포넌트 이름을 DefaultGNB로 변경
- 로그인 버튼 삭제

dabinchiii committed:
feat(search bar): 검색창 컴포넌트 기능 추가

- 입력한 텍스트를 초기화하는 버튼 추가
- 검색 버튼 클릭 시 검색 결과 페이지로 이동
- useNavigate의 state로 검색할 키워드 전달
- 상세 검색 버튼 추가 (기능 추후 구현)
- focus 되면 그림자 색상 변경됨

dabinchiii committed:
feat(GNB): GNB에 검색창 추가

dabinchiii committed:
feat(layout): 검색 페이지를 위한 레이아웃 추가

dabinchiii committed:
feat(api): 북마크 조회, 북마크 검색 api 호출 함수 추가

SeonghyeonKim committed:
fix: 북마크 이동 시 에러 메세지 수정

SeonghyeonKim committed:
fix: 에러코드에 맞게 메세지 수정

dabinchiii committed:
Merge pull request #144 from SeonghyeonKim/feat/bookmark-edit

북마크 수정, 삭제 기능 추가

dabinchiii committed:
Merge pull request #145 from SeonghyeonKim/fix/bookmark-move

북마크 이동 후 생기는 현상 수정

dabinchiii committed:
feat(loader): 비동기 데이터를 사용하는 컴포넌트에 Suspense 적용

dabinchiii committed:
feat(api): 북마크 검색 api 호출 함수에 page 매개변수 추가

SeonghyeonKim committed:
fix: 재렌더링 해결

dabinchiii committed:
fix(card): 이미지가 잘리는 에러 해결

SeonghyeonKim committed:
fix: 공유 카테고리 화면의 카드 이미지 수정

SeonghyeonKim committed:
Merge pull request #146 from SeonghyeonKim/fix/re-render

북마크 수정, 이동, 삭제 시 전체 북마크 리스트가 재 렌더링 되도록 설정

SeonghyeonKim committed:
Merge pull request #147 from SeonghyeonKim/fix/image

공유 카테고리 화면의 이미지 수정

SeonghyeonKim committed:
Merge pull request #148 from Step3-kakao-tech-campus/weekly

배포 테스트를 위한 PR

SeonghyeonKim committed:
Merge pull request #149 from Step3-kakao-tech-campus/develop

배포 테스트를 위한 PR

SeonghyeonKim committed:
fix: props 추가

SeonghyeonKim committed:
Merge pull request #150 from SeonghyeonKim/fix/bug

fix: props 추가

dabinchiii committed:
feat(search): 상세 검색 기능 구현

dabinchiii committed:
feat(Searchbar): 검색창에 입력 후 엔터키 누르면 검색 버튼 클릭되는 기능

dabinchiii committed:
feat(page): 검색 결과 페이지 구현

dabinchiii committed:
feat: prettier 적용

dabinchiii committed:
feat(search bar): 검색창의 상세 검색 버튼을 항상 보이도록 하고 클릭 시
동작 구현

dabinchiii committed:
fix(search): 상세 검색 box를 닫아도 border가 남아있던 문제 해결

dabinchiii committed:
feat: suspense 처리

SeonghyeonKim committed:
Merge pull request #151 from dabinchiii/feat/search

북마크 검색 기능 구현

SeonghyeonKim committed:
Merge pull request #152 from Step3-kakao-tech-campus/weekly

배포 테스트를 위한 PR

SeonghyeonKim committed:
Merge pull request #153 from Step3-kakao-tech-campus/develop

배포 테스트를 위한 PR

dabinchiii committed:
feat(tag): 태그 컴포넌트에 title 추가하여 마우스 올리면 값 보이도록 변경

dabinchiii committed:
feat(card): NonDraggableCard의 스타일 수정

- 제목과 태그가 카드에서 튀어나오지 않도록 변경
- hover시 border와 shadow 변경되어 강조

dabinchiii committed:
fix(share link): 암호화 ID 추출 함수 버그 해결

dabinchiii committed:
Merge pull request #13 from dabinchiii/feat/bookmark-add/image

Feat/bookmark add/image

SeonghyeonKim committed:
feat: 애니메이션 추가

SeonghyeonKim committed:
feat: 애니메이션이 적용되는 모달창 제작

SeonghyeonKim committed:
feat: 첫 시작 시 시작페이지 표시

SeonghyeonKim committed:
feat: 시작페이지 제작

SeonghyeonKim committed:
Merge pull request #154 from dabinchiii/fix

Fix

dabinchiii committed:
Merge pull request #155 from SeonghyeonKim/feat/start-page

시작페이지 프로토타입 제작

dabinchiii committed:
fix(api): 암호화된 ID를 url encode하여 보내도록 변경

dabinchiii committed:
fix(share link): 공유 링크에서 암호화 ID 추출 에러 해결

dabinchiii committed:
fix(share): 공유 링크로 추가하기의 에러 해결

- 워크스페이스를 선택하지 않았을 경우의 처리 방식 변경
- 추출 함수에서 url을 url 인코딩하여 처리 후 디코딩한 값 리턴하도록 변경

dabinchiii committed:
Merge pull request #14 from dabinchiii/feat/bookmark-add/image

Feat/bookmark add/image

dabinchiii committed:
fix(share): 공유 링크로 추가하기 에러 해결

- url로 변환하던 방식을 다시 문자열 비교로 변경

dabinchiii committed:
Merge pull request #15 from dabinchiii/feat/bookmark-add/image

fix(share): 공유 링크로 추가하기 에러 해결

dabinchiii committed:
Merge pull request #156 from dabinchiii/fix

공유 링크 관련 에러 해결

SeonghyeonKim committed:
Merge pull request #157 from Step3-kakao-tech-campus/weekly

배포 테스트를 위한 PR

SeonghyeonKim committed:
Merge pull request #158 from Step3-kakao-tech-campus/develop

배포 테스트를 위한 PR

dabinchiii committed:
feat(Modal): 이미지 선택을 위한 컴포넌트 구현

- URL 또는 파일 형식의 이미지를 선택할 수 있는 컴포넌트 구현

SeonghyeonKim committed:
feat: 파비콘, 로고 변경

SeonghyeonKim committed:
feat: 서비스 타이틀 변경

SeonghyeonKim committed:
feat: hover 스타일 설정

SeonghyeonKim committed:
feat: hover 스타일 설정

SeonghyeonKim committed:
feat: 시작페이지 마지막 모달 버튼 추가

SeonghyeonKim committed:
feat: 시작페이지 첫 모달 이동 핸들러 추가

SeonghyeonKim committed:
fix: 메인페이지에서 로그인페이지로 이동 삭제

SeonghyeonKim committed:
fix: 토큰 확인 후 로그인/아웃 버튼 렌더링

dabinchiii committed:
feat(util): 이미지 파일을 base64로 인코딩하는 함수 구현

SeonghyeonKim committed:
feat: 최근 등록한 북마크 리스트 조회 api 작성

SeonghyeonKim committed:
feat: 로그인 후 시작페이지 제작

SeonghyeonKim committed:
feat: 로그인 여부로 시작페이지 렌더링

dabinchiii committed:
Merge pull request #162 from SeonghyeonKim/fix/structure

페이지 스타일, 구조 변경

dabinchiii committed:
feat(api): 북마크 이미지 수정 api 호출 함수 추가

dabinchiii committed:
feat(Modal): 북마크 추가 모달에 이미지 선택 기능 추가

- 이미지 선택 여부를 체크박스로 입력받음
- 이미지 선택 체크박스가 checked가 되면, 이미지 URL과 이미지 파일 중
  하나를 선택 가능한 영역이 표시됨
- 이미지 파일 선택 시 base64로 인코딩되어 저장됨

dabinchiii committed:
feat(utils): 문자열을 태그 리스트로 변환하는 함수 추가

dabinchiii committed:
feat(context menu): 북마크 context menu에 북마크 이미지 변경하기 메뉴
추가

SeonghyeonKim committed:
fix: AddCard의 props 변경

SeonghyeonKim committed:
feat: 모달에 핸들러 추가

dabinchiii committed:
fix(api): 북마크 이미지 변경 api 호출 함수의 오타 수정

dabinchiii committed:
feat(Modal): 이미지 URL input에 placeholder 추가

dabinchiii committed:
feat(Modal): 북마크 이미지 변경 모달 구현 및 오타 수정

dabinchiii committed:
Merge pull request #164 from SeonghyeonKim/fix/render

북마크 추가 이후 렌더링하도록 수정

dabinchiii committed:
Merge pull request #16 from dabinchiii/feat/bookmark-add/image

Feat/bookmark add/image

dabinchiii committed:
Merge branch 'weekly' of https://github.com/Step3-kakao-tech-campus/Team9_FE into newWeekly

dabinchiii committed:
fix(Modal): 북마크 이미지 수정 후 데이터 refetch 되도록 변경

dabinchiii committed:
refactor(Modal): global modal의 모달 컴포넌트들을 상수화

- BookmarkImageEditModal 상수를 BookmarkImageUpdateModal로 변경

dabinchiii committed:
feat(page): not found 페이지 추가

dabinchiii committed:
fix(page): 404 not found 페이지 path를 수정

SeonghyeonKim committed:
Merge pull request #165 from dabinchiii/feat/bookmark-add/image

북마크에 이미지 추가/수정하는 기능 구현

dabinchiii committed:
fix(utils): base64 변환 함수에서 이미지 형식을 나타내는 부분을 제거한 값
반환하도록 수정

SeonghyeonKim committed:
Merge branch 'weekly' into newWeekly

dabinchiii committed:
Merge branch 'weekly' of https://github.com/Step3-kakao-tech-campus/Team9_FE into newWeekly

SeonghyeonKim committed:
Merge pull request #166 from dabinchiii/newWeekly

404 페이지 추가, global 모달 리팩토링

dabinchiii committed:
Merge branch 'weekly' of https://github.com/Step3-kakao-tech-campus/Team9_FE into newWeekly

dabinchiii committed:
refactor: 문자열에서 태그 리스트 추출하는 함수 적용

SeonghyeonKim committed:
Merge pull request #167 from Step3-kakao-tech-campus/weekly

배포 테스트를 위한 PR

SeonghyeonKim committed:
Merge pull request #168 from Step3-kakao-tech-campus/develop

배포 테스트를 위한 PR

dabinchiii committed:
style(page): 검색 결과 페이지 디자인 변경

SeonghyeonKim committed:
fix: 첫 렌더링에만 리스트를 호출하도록 변경

dabinchiii committed:
style: 화면이 확대되어도 북마크 그리드 사이의 공백이 커지지 않도록 변경

dabinchiii committed:
feat(error): 북마크 추가 모달에서 에러 캐칭 변경

dabinchiii committed:
Merge pull request #169 from SeonghyeonKim/fix/list

첫 렌더링에만 리스트를 호출하도록 변경

dabinchiii committed:
Merge branch 'weekly' of https://github.com/Step3-kakao-tech-campus/Team9_FE into newWeekly

SeonghyeonKim committed:
Merge pull request #170 from dabinchiii/newWeekly

Feat

SeonghyeonKim committed:
Merge pull request #171 from Step3-kakao-tech-campus/weekly

배포 테스트를 위한 PR

SeonghyeonKim committed:
Merge pull request #172 from Step3-kakao-tech-campus/develop

배포 테스트를 위한 PR

dabinchiii committed:
style(page): not found 페이지 디자인 변경

dabinchiii committed:
feat(page): 403 forbidden 페이지 추가

dabinchiii committed:
fefactor: public 이미지를 상수로 저장

kyoungsik-dev commented on link-namu/src/pages/NotionRedirectPage.jsx:
```diff
@@ -0,0 +1,28 @@
+  const parentWindow = window.opener;
+  const currentUrl = window.location.href;
+
+  const getCodeFromUrl = (url) => {
+    const regex = new RegExp("[?&]code(=([^&#]*)|&|#|$)");
+    const results = regex.exec(url);
+    if (!results) return null;
+    if (!results[2]) return "";
+    return decodeURIComponent(results[2].replace(/\+/g, " "));
+  };
```
이런 코드들은 어떤 동작을 수행하는지 이해하기 어려울 것 같아요.
이럴 때에는 유지보수 차원에서 주석을 적극적으로 활용해주시는 게 좋습니다.

dabinchiii committed:
style: GNB의 로고 변경

kyoungsik-dev reviewed (approved):
코드 양이 엄청나네요 ㅎㅎ
전반적으로 코드를 깔끔하게 잘 작성해주셨네요.
마무리만 잘 해주시면 될 것 같아요.
지난 두달간 수고 많으셨습니다.
고생하셨어요 :)

dabinchiii committed:
feat: 404 응답 시 not found 페이지로 이동하는 기능

dabinchiii committed:
fix(toast): 타입이 지정되지 않은 토스트 수정

dabinchiii committed:
style: 메인 페이지에서 breadcrumbs와 북마크 그리드 수직 경계선을
동일하게 변경

dabinchiii committed:
fix: 공백만 있는 문자열일 때 태그 리스트에 공백 문자열이 존재하는 에러
해결

- 공백 문자열일 경우, 빈 배열을 반환하도록 stringToTags 함수 수정

dabinchiii committed:
fix(Modal): 공유 링크로 추가하기 모달 에러 해결

- 공유 링크 input 값이 변할 때마다 코드를 추출하여 에러 토스트가 계속
  뜨던 에러 해결

dabinchiii committed:
fix: 오타 수정

dabinchiii committed:
feat: 403 페이지 이동 기능

dabinchiii committed:
feat: 로고 배경 제거

SeonghyeonKim committed:
Merge pull request #177 from dabinchiii/newWeekly

에러 수정

SeonghyeonKim committed:
Merge pull request #178 from Step3-kakao-tech-campus/weekly

배포 테스트를 위한 PR

SeonghyeonKim committed:
Merge pull request #179 from Step3-kakao-tech-campus/develop

배포 테스트를 위한 PR

dabinchiii committed:
feat: favicon 배경 제거

dabinchiii committed:
style: 로그인 페이지 디자인 변경

SeonghyeonKim committed:
fix: 태그의 크기가 컴포넌트를 넘지 않도록 수정

dabinchiii committed:
fix(search): 상세 검색 값 유효성 검사

- 북마크 제목 또는 태그가 있어야 요청을 보냄

dabinchiii committed:
fix(search): 검색창의 입력값이 없으면 요청 보내지 않음

dabinchiii committed:
feat: 카카오 모달 수정

dabinchiii committed:
feat

dabinchiii committed:
Merge pull request #180 from SeonghyeonKim/fix/last-fix

태그의 크기가 컴포넌트를 넘지 않도록 수정

SeonghyeonKim committed:
Merge pull request #181 from dabinchiii/newWeekly

New weekly

SeonghyeonKim committed:
Merge pull request #182 from Step3-kakao-tech-campus/weekly

배포 테스트를 위한 PR

SeonghyeonKim committed:
Merge pull request #183 from Step3-kakao-tech-campus/develop

배포 테스트를 위한 PR

SeonghyeonKim committed:
fix: 오타 수정

SeonghyeonKim committed:
fix: 기본 GNB에 props 추가

SeonghyeonKim committed:
fix: 엑세스 토큰이 없는 경우 수정

SeonghyeonKim committed:
Merge pull request #184 from SeonghyeonKim/fix/last-fix

엑세스 토큰이 없는 경우, 접근 못하게 컴포넌트 제거

SeonghyeonKim committed:
Merge pull request #185 from Step3-kakao-tech-campus/weekly

배포 테스트를 위한 PR

SeonghyeonKim committed:
Merge pull request #186 from Step3-kakao-tech-campus/develop

배포 테스트를 위한 PR

dabinchiii committed:
Update README.md

dabinchiii committed:
Update README.md

SeonghyeonKim committed:
Create LICENSE

SeonghyeonKim committed:
Update README.md

SeonghyeonKim committed:
fix: 북마크 생성 시 http 추가

SeonghyeonKim committed:
fix: 오타 수정

SeonghyeonKim committed:
Merge pull request #187 from SeonghyeonKim/fix/last-fix

핫픽스

SeonghyeonKim committed:
Update README.md

SeonghyeonKim committed:
Update README.md

SeonghyeonKim committed:
Update Dockerfile

SeonghyeonKim committed:
Update README.md

SeonghyeonKim committed:
fix: 리프레시 토큰으로 수정

SeonghyeonKim committed:
Merge pull request #188 from SeonghyeonKim/fix/last-fix

핫픽스

SeonghyeonKim committed:
fix: 엑세스 토큰 저장

SeonghyeonKim committed:
Merge pull request #189 from SeonghyeonKim/fix/last-fix

핫픽스

SeonghyeonKim committed:
fix: 인스턴스를 쿠키에서 가져오도록 변경

SeonghyeonKim committed:
fix: 수정

SeonghyeonKim committed:
Merge pull request #190 from SeonghyeonKim/fix/last-fix

핫픽스

dabinchiii committed:
fix: 로그아웃 시 액세스 토큰도 삭제

dabinchiii committed:
Merge pull request #192 from dabinchiii/newWeekly

fix: 로그아웃 시 액세스 토큰도 삭제

SeonghyeonKim committed:
fix: 최근 등록 북마크 페이지네이션으로 교체

SeonghyeonKim committed:
Merge pull request #193 from SeonghyeonKim/fix/last-fix

최근 등록 북마크 페이지네이션으로 교체

SeonghyeonKim committed:
fix: api 수정에 맞춰서 수정

SeonghyeonKim committed:
Merge pull request #194 from SeonghyeonKim/fix/last-fix

api 수정에 맞춰서 수정

SeonghyeonKim committed:
fix: 설명이 2줄 나오도록 변경

SeonghyeonKim committed:
feat: 워크스페이스 출처 추가

SeonghyeonKim committed:
feat: 링크 출처에 따라 아이콘 변경

SeonghyeonKim committed:
fix: 최근 북마크 api 수정

SeonghyeonKim committed:
Merge pull request #195 from SeonghyeonKim/fix/last-fix

회의 내용 수정

SeonghyeonKim committed:
fix: 스타일 변경

SeonghyeonKim committed:
fix: 로그아웃 시 메인페이지로 이동

SeonghyeonKim committed:
Merge branch 'master' into fix/last-fix

SeonghyeonKim committed:
Merge pull request #196 from SeonghyeonKim/fix/last-fix

로그아웃 시 메인 페이지로 이동

dabinchiii committed:
Merge pull request #17 from Step3-kakao-tech-campus/master

master

dabinchiii committed:
feat: prettier

dabinchiii committed:
feat: 토큰 재발급 로직을 커스텀 훅을 사용하도록 변경

SeonghyeonKim committed:
fix: 카드 형식 수정

SeonghyeonKim committed:
Merge branch 'fix/last-fix' of https://github.com/SeonghyeonKim/Team9_FE into fix/last-fix

SeonghyeonKim committed:
Merge pull request #197 from SeonghyeonKim/fix/last-fix

카드 형식 변경

SeonghyeonKim committed:
fix: 카드 디자인 변경

SeonghyeonKim committed:
feat: 임시보관함 컨텐츠 닫기 기능 추가

SeonghyeonKim committed:
Merge pull request #198 from SeonghyeonKim/fix/last-fix

다른 카드의 디자인 변경, 닫기 기능 추가

dabinchiii committed:
feat(token): token을 관리하는 로직을 하나의 파일에서 관리하도록 변경

- 쿠키에 RefreshToken과 AccessToken을 저장
- 토큰의 만료 기간이 지나면 쿠키에서 삭제되도록 만료 기간을 지정함
- 토큰을 쿠키에 저장/삭제하는 함수
- 쿠키에 저장된 토큰 값을 가져오는 함수
- 쿠키에 토큰이 저장되어 있는지 확인하는 함수

dabinchiii committed:
feat(store): 토큰 값을 쿠키에서 가져오도록 변경

dabinchiii committed:
feat(GNB): default GNB에 로그인 버튼 표시 여부를 선택할 수 있도록 함

dabinchiii committed:
feat(GNB): 쿠키에 직접 접근하지 않고 함수를 통해 접근하도록 변경

dabinchiii committed:
feat(signin): 쿠키에 직접 접근하지 않고 함수를 통해 접근하도록 변경

dabinchiii committed:
feat: 시작 페이지를 메인 페이지에서 분리

dabinchiii committed:
feat(token): access token 재발급 로직 변경

- access token이 만료됐는데 요청을 보내서 401 에러를 응답받으면, 토큰을
  재발급 후 다시 요청 보내도록 변경

dabinchiii committed:
Merge pull request #199 from Step3-kakao-tech-campus/master

master를 weekly에 덮어쓰기

dabinchiii committed:
Merge pull request #200 from dabinchiii/feat/web-accessibility

access token 재발급 로직 변경 및 시작 페이지를 메인 페이지에서 분리

dabinchiii committed:
Merge branch 'master' of https://github.com/Step3-kakao-tech-campus/Team9_FE into feat/web-accessibility

dabinchiii committed:
Merge branch 'weekly' of https://github.com/Step3-kakao-tech-campus/Team9_FE into feat/web-accessibility

dabinchiii committed:
feat: 로더에 title 추가

dabinchiii committed:
refactor: 버튼 기능을 하는 컴포넌트를 버튼으로 변경

dabinchiii committed:
feat: prettier

dabinchiii committed:
feat(aria): 모달이 열릴 때, app 컴포넌트를 aria-hidden으로 설정

dabinchiii committed:
feat: 사이드바 타일에 title 추가

dabinchiii committed:
feat: ModalTextInput에 placeholder 추가

dabinchiii committed:
feat: 워크스페이스/카테고리 아이템의 title에 정보 추가

dabinchiii committed:
feat: title 추가

dabinchiii committed:
feat: title 추가

dabinchiii committed:
feat: breadcrumbs의 max width 설정

dabinchiii committed:
feat: SharedBookmarkGrid의 카드들에 key 속성 추가

dabinchiii committed:
feat: 워크스페이스 추가 버튼 호버 시 색상 변경

dabinchiii committed:
feat: title 추가

dabinchiii committed:
style: tag 스타일 변경

dabinchiii committed:
Merge pull request #201 from dabinchiii/feat/web-accessibility

Feat/web accessibility

Train0303 committed:
Merge pull request #202 from Step3-kakao-tech-campus/weekly

웹 접근성 및 토큰 관리 방식 변경

SeonghyeonKim committed:
fix: 북마크 이동 시 오류 수정

SeonghyeonKim committed:
Merge pull request #203 from SeonghyeonKim/fix/last-fix

북마크 이동 시 오류 수정

SeonghyeonKim committed:
fix: 제목 전체 표시하도록 변경

SeonghyeonKim committed:
Merge branch 'master' into fix/last-fix

SeonghyeonKim committed:
Merge pull request #204 from SeonghyeonKim/fix/last-fix

hover 시 제목 전체 표시하도록 변경

SeonghyeonKim committed:
fix: 제목 전체가 나오게 변경

SeonghyeonKim committed:
Merge branch 'fix/last-fix' of https://github.com/SeonghyeonKim/Team9_FE into fix/last-fix

SeonghyeonKim committed:
fix: style

SeonghyeonKim committed:
Merge pull request #205 from SeonghyeonKim/fix/last-fix

hover 시 제목 전체가 나오도록 변경
